app/views/shared/_notification.html.erb

```diff
@@ -1,45 +1,45 @@
-<%# locals: (type: "success", content: { title: '', body: ''}, action: { label:'' , url:'' }, options: { auto_dismiss: true }) -%>
+<%# locals: (message:, type: "notice", id: nil, **_opts) %>
```

Comment:
**_opts are needed here since turbo rails automatically passes the model object on any `broadcast_<verb>_to` methods, and we construct this partial to send via streams sometimes.